.gitattributes

```diff
@@ -2,10 +2,13 @@
 # https://www.kernel.org/pub/software/scm/git/docs/gitattributes.html
 
 # Ignore all test and documentation with "export-ignore".
+/.github            export-ignore
 /.gitattributes     export-ignore
 /.gitignore         export-ignore
 /.travis.yml        export-ignore
 /phpunit.xml.dist   export-ignore
 /.scrutinizer.yml   export-ignore
 /tests              export-ignore
 /.editorconfig      export-ignore
+/CHANGELOG.md       export-ignore
+/.php-cs-fixer.dist.php export-ignore
```

Comment:
Please align export-ignore so that it is in the same position on every line. Do this for each of the PRs you sent. Thanks!

Reply:
Smaller filename